### ISSUE TITLE
validating output files without an accompanying metadata file

### ISSUE THREAD

**Issue by elray1** [hubverse-org/hubValidations#138]
If I try to validate a model output file without an accompanying metadata file, I get an early exit before most checks are performed:

```
> hubValidations::validate_submission(
+     hub_path = ".",
+     file_path = file.path("example-submission", paste0(reference_date, "-example-submission.csv")),
+     skip_submit_window_check = TRUE
+ )

── FluSight-forecast-hub ────

✔ [valid_config]: All hub config files are valid.

── 2024-11-16-example-submission.csv ────

✔ [file_exists]: File exists at path model-output/example-submission/2024-11-16-example-submission.csv.
✔ [file_name]: File name "2024-11-16-example-submission.csv" is valid.
✔ [file_location]: File directory name matches `model_id` metadata in file name.
✔ [round_id_valid]: `round_id` is valid.
✔ [file_format]: File is accepted hub format.
ⓧ [metadata_exists]: Metadata file does not exist at path model-metadata/example-submission.yml or
  model-metadata/example-submission.yaml.
```

I would think that most of the remaining checks would not depend on the existence of a model metadata file.  Is there any way to get around this?

**Comment by zkamvar** [comment]
Would the proposal of a `validate_model_tbl()` function in #130 help? This would allow you to circumvent the need to write the table to a file if you are iterating.

**Comment by elray1** [comment]
That would certainly help!  I am also wondering if this is something where that check is resulting an error that results in halting the validations process, where we could change it to a non-halting failure.

**Comment by annakrystalli** [comment]
Agreed, early return is a bit excessive for this. Fixed in #144 

As an aside, you could use `validate_model_data()` to just validate file contents.
